File: app/src/main/java/com/kavir/ali/kavir_msg/models/SubCategoryModel.java
package com.kavir.ali.kavir_msg.models;

import com.google.gson.annotations.SerializedName;

public class SubCategoryModel {
    @SerializedName("id")
    private int id;

    @SerializedName("slug")
    private String slug;

    @SerializedName("title")
    private String title;

    @SerializedName("description")
    private String description;

    @SerializedName("post_count")
    private int post_count;

    @SerializedName("parent")
    private int parent;

    public SubCategoryModel(int id, String slug, String title, String description, int post_count, int parent) {
        this.id = id;
        this.slug = slug;
        this.title = title;
        this.description = description;
        this.post_count = post_count;
        this.parent = parent;
    }

    public SubCategoryModel() {

    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public int getPost_count() {
        return post_count;
    }

    public void setPost_count(int post_count) {
        this.post_count = post_count;
    }

    public int getParent() {
        return parent;
    }

    public void setParent(int parent) {
        this.parent = parent;
    }
}

File: app/src/main/java/com/kavirelectronic/ali/kavir_info/models/SubCategoryModel.kt
package com.kavirelectronic.ali.kavir_info.models

import com.google.gson.annotations.SerializedName

class SubCategoryModel {
    @SerializedName("id")
    var id = 0

    @SerializedName("slug")
    var slug: String? = null

    @SerializedName("title")
    var title: String? = null

    @SerializedName("description")
    var description: String? = null

    @SerializedName("post_count")
    var post_count = 0

    @SerializedName("parent")
    var parent = 0

    constructor(id: Int, slug: String?, title: String?, description: String?, post_count: Int, parent: Int) {
        this.id = id
        this.slug = slug
        this.title = title
        this.description = description
        this.post_count = post_count
        this.parent = parent
    }

    constructor() {}

}